### DISCUSSION TITLE
SubSquid as a default indexer

### DISCUSSION THREAD

**Discussion by vikiival** [kodadot/nft-gallery#2929] [Q&A] [answered]
Currently, we are using a combination of subquery and subsquid which requires a lot of hacks and the app does bring a ton of compatibility issues.

The idea is to use subsquid as a default indexer for everything. 

Feel free to discuss what's your opinion

**Comment by kkukelka** [comment]
Is SubSquid stable in its current form? I remember you saying recently that its still in "alpha-stage" (i believe you mentioned it in your graphql tutorial presentation)

**Reply by vikiival** [reply]
Currently for Basilisk SSQ is the default one. I plan to do the same for statemine too. 

I am just thiking aloud to use SSQ for Kusama NFTs. 

Longterm plan is to support rubick for both RMRK / KodaProtocol. 

magick should stay for RMRK Classic

**Comment by yangwao** [comment]
I thought we could have an abstract api and the end user dev doesn't need to care which endpoint is using for particular action?
Was it way too complex?

**Reply by vikiival** [reply]
This should be a task for UniQuery (previously Kodapi)

but some queries are extremly hard to write them into single query builder

**Comment by vikiival** [comment]
Bump this becomes urgent imo.
Once the kodadot/rubick#112 is merged happy to make it as main

**Comment by vikiival** [comment, marked as answer]
Meanwhile it was already merged as #3640 and new 🔥🦑 is working hard kodadot/rubick#117
